tools/semgrep/ci/fmt-printf.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: fmt.Printf
+    languages: [go]
+    message: fmt.Printf is forbidden outside of cmd and test files
+    pattern: fmt.Printf
+    severity: ERROR
+    paths:
+      exclude:
+        - "*test*.go"
```

Comment:
Don't go test files have to end in `_test.go`? Seems like this will catch any file with `test` in the name, rather than actual test files.

Reply:
:+1: